openhands/events/serialization/event.py

```diff
@@ -36,6 +36,9 @@
 
 
 def event_from_dict(data) -> 'Event':
+    if 'images_urls' in data:
+        data['image_urls'] = data.pop('images_urls')
```

Comment:
This isn't going to work, sorry, because it's not directly in `data`, it's in `data['args']`. And maybe a better place for this is [here](https://github.com/All-Hands-AI/OpenHands/blob/975e75531d708cbdd3f9c284840588cafc40de34/openhands/events/serialization/action.py#L63), where we have another compatibility tweak.

Reply:
Updated.